assets/js/templates/revert-button/index.tsx

```diff
@@ -140,12 +140,7 @@ let currentTemplateId: string | undefined;
 subscribe( () => {
 	const previousTemplateId = currentTemplateId;
 	const store = select( 'core/edit-site' );
-
-	if ( ! isSiteEditorPage( store ) ) {
-		return;
-	}
-
-	currentTemplateId = store?.getEditedPostId();
+	currentTemplateId = store.getEditedPostId();
```

Comment:
It's safe to assume that this component will always be present in a site-editor environment?

Reply:
You are right, it's better to not make this assumption, updated 👍 !